### PR TITLE
[12.0] [IMP] product_pricelist_direct_print: xlsx and new option to print defined products

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT27988

@carlosdauden @sergio-teruel review this

### CODE REVIEW

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Can you increase the coverage? Is it possible?

**Comment by CarlosRoca13** [comment]
@sergio-teruel done

**Review by carlosdauden — APPROVED** [review]

File: product_pricelist_direct_print/report/product_pricelist_xlsx.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class ProductPricelistXlsx(models.AbstractModel):
+    _name = 'report.product_pricelist_direct_print.product_pricelist_xlsx'
+    _inherit = 'report.report_xlsx.abstract'
```
Comment (sergio-teruel) [inline product_pricelist_direct_print/report/product_pricelist_xlsx.py:8]: Add _description attribute

**Review by sergio-teruel — COMMENTED** [review]
Why these tow columns is on right? Can yoy move it more on the left?
image

**Comment by CarlosRoca13** [comment]
@sergio-teruel changes done

**Comment by sergio-teruel** [comment]
Ok. Thanks

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**